src/parsers/parsers.js

```diff
@@ -1,11 +1,47 @@
+// eslint-disable-next-line import/no-cycle
+import { defaultMaterialConfig } from "../material";
```

Comment:
Moving the xyzToPoscar function fixed the xyz circular dependency, but created one for defaultMaterialConfig. Previously this circular dependency was noted inside `parsers/xyz.js`